src/QMCWaveFunctions/tests/test_cuSolverInverter.cpp

```diff
@@ -0,0 +1,78 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2022 QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#include "catch.hpp"
+
+#include "Configuration.h"
+#include "Platforms/CPU/SIMD/aligned_allocator.hpp"
+#include "QMCWaveFunctions/Fermion/DiracMatrix.h"
+#include "QMCWaveFunctions/Fermion/cuSolverInverter.hpp"
+#include "checkMatrix.hpp"
+#include "createTestMatrix.h"
+
+namespace qmcplusplus
+{
+
+using LogValueType = std::complex<double>;
+
+TEMPLATE_TEST_CASE("cuSolverInverter", "[wavefunction][fermion]", double, float)
+{
+#ifdef QMC_COMPLEX
+  using FullPrecValueType = std::complex<double>;
+  using ValueType         = std::complex<TestType>;
```

Comment:
Where is TestType defined?
Please use the latest type convention. Value instead of ValueType.

Reply:
TestType comes from Catch.  I'm using a TEMPLATE_TEST_CASE to test float and double versions in the same test case. I added a comment about it.